news/4111.bugfix

```diff
@@ -0,0 +1 @@
+Fix ``pip install --target`` by making ``--target``, ``--user``, and ``--prefix`` mutually exclusive.
```

Comment:
I would phrase this as:

    pip install `--target` is now mutually exclusive with `--user` and `--prefix`.

Reply:
Done.